views/new.erb

```diff
@@ -0,0 +1,4 @@
+<form action="/" method="post">
```

Comment:
How do you get to this page? Do you have to type '/new' in the url? If so, maybe a button on the homepage could take you here instead?